include/fi_file.h

```diff
@@ -46,7 +47,6 @@ extern "C" {
 
 int fi_read_file(const char *dir, const char *file, char *buf, size_t size);
 int fi_poll_fd(int fd, int timeout);
-int fi_fd_nonblock(int fd);
```

Comment:
This is pretty minor and I don't care that much about it, it just struck me as an odd change.  I'd consider changing the Windows version to not be a `static inline` (i.e., move the implementation to `src/windows/osd.c`), since it's unlikely to be in the critical path anywhere.